test/unit/krane/template_sets_test.rb

```diff
@@ -16,7 +16,8 @@ def test_empty_template_sets_directory_is_invalid
     Dir.mktmpdir("empty_dir") do |dir|
       template_sets = template_sets_from_paths(dir)
       expected = [
-        "Template directory #{dir} does not contain any valid templates",
+        "Template directory #{dir} does not contain any valid templates " \
+          "(supported suffixes: .yml.erb, .yml, .yaml, .yaml.erb, or secrets.ejson)",
       ]
       assert_equal(template_sets.validate, expected)
```

Comment:
if `render_erb: false`, do we still say that *.erb suffixes are supported?

Reply:
Nope, we adjust the list of supported suffixes as needed.